snapcraft/cli/_options.py

```diff
@@ -65,4 +65,4 @@ def get_project_options(**kwargs):
         target_deb_arch=kwargs.pop('target_arch'),
     )
 
-    return ProjectOptions(**project_args)
+    return ProjectOptions(**project_args, **kwargs)
```

Comment:
That's what the code did. But since it wasn't clear enough, I removed the second dict.